platform/admin/modules/admin-docs-ui/src/main/java/org/codice/ui/admin/docs/DocsSparkApplication.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+
+package org.codice.ui.admin.docs;
+
+import static spark.Spark.get;
+import static spark.Spark.staticFiles;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Collection;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.filefilter.NameFileFilter;
+import org.apache.commons.io.filefilter.TrueFileFilter;
+
+import spark.servlet.SparkApplication;
+
+public class DocsSparkApplication implements SparkApplication {
+
+    public static final String DOCS_ROOT_DIR = String.format("%s/documentation/",
```

Comment:
Does this work on Windows?  You should use `Paths.get` instead of `String.format` when working with directory paths.

https://docs.oracle.com/javase/7/docs/api/java/nio/file/Paths.html